### PR TITLE
Hello rails back-end: Set up

### PR DESCRIPTION
**Pull request by amare1990**
**In this project, I:**

- [x] Followed Gitflow.
- [x] Created a rails api which is used as API end points for the react front-end app.
- [x] Created Greeting model that allows admins to store greeting message in the postgreql database table.
- [x] Implemented the functionality of selecting one message and fetching from the five messages stored in the table.
- [x] Created json end point for the selected message.
- [x] Checked rubocop linter bugs.
- [x] Updated the Microverse README.md template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,44 @@
...
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Hello Rails Back-end <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+**Hello Rails Back-end** is a rails api that randomly selects one message from the postgresql database table and creates API end point to send JSON message data. It is meant to exercise setting up the rails back-end and connect to the react-front-end app as two apps.
 
-**[your_project__name]** is a...
+**Link to the react-front-end**
+- [React-front-end-repo](https://github.com/amare1990/hello-react-front-end)
```
Comment (tugsem) [inline README.md:25]: - [ ] Kindly add the link to this repo in your frontend repo's readme as well.

**Review by tugsem — COMMENTED** [review]
## Invalid Code Review Request ❌ 

Hi @amare1990, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h1>You have submitted a project, that doesn't provide a link to your backend in your frontend repo.</h2>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (amare1990) [reply README.md:25]: Hi @tugsem . Thank you for reviewing my project. Can you check out the changes I made now?

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Amare,  

Your project is complete, great job for nailing it! 💎  There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 👍
💠 React frontend connect correctly Rails API backend!
💠 Professional ReadMe!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**